pkgs/development/python-modules/pysam/default.nix

```diff
@@ -8,35 +8,36 @@
 , htslib
 , libdeflate
 , xz
-, pytest
+, pytestCheckHook
 , samtools
 , zlib
 }:
 
 buildPythonPackage rec {
   pname   = "pysam";
-  version = "0.20.0";
+  version = "0.21.0";
 
   # Fetching from GitHub instead of PyPi cause the 0.13 src release on PyPi is
   # missing some files which cause test failures.
   # Tracked at: https://github.com/pysam-developers/pysam/issues/616
   src = fetchFromGitHub {
     owner = "pysam-developers";
-    repo = "pysam";
-    rev = "refs/tags/v${version}";
```

Comment:
Please revert that change. Using `refs/tags` helps to avoid branch/tag which could happen and are annoying for bulk updates.

Reply:
Changed as requested